ewatercycle/util.py

```diff
@@ -87,6 +86,26 @@ def lat_lon_boundingbox_to_variable_indices(model, variable, lat_min, lat_max, l
     return np.array(output)
 
 
+def find_closest_point(lonitudes: Iterable[float], latitudes: Iterable[float], longitude: float, latitude: float) -> Tuple[np.ndarray, int]:
```

Comment:
```suggestion
def find_closest_point(longitudes: Iterable[float], latitudes: Iterable[float], longitude: float, latitude: float) -> Tuple[np.ndarray, int]:
```

or looneytunes?

also below

Reply:
Maybe it would be good to make a bit more distinct names for grid_longitudes and point_longitude, for example.

Reply:
The arguments are refactored, and typo is fixed :-)